### ISSUE TITLE
Default template

### ISSUE THREAD

**Issue by rochana-atapattu** [epwalsh/obsidian.nvim#323]
### 📚 The doc issue

There is a default template added when you run :ObsidianNew which is not properly documented and different from apps behavior.
 
<img width="417" alt="Screenshot 2024-01-12 at 19 48 45" src="https://github.com/epwalsh/obsidian.nvim/assets/31321610/83259ac2-2100-441b-a0f9-31839f841f6c">

**Comment by bcl1713** [comment]
This isn't a template per se, as much as its the formatter adding the yaml frontmatter to your file which you can control with the configuration.

`disable_frontmatter = true`

**Comment by epwalsh** [comment]
Yes @bcl1713 is right, although I can understand the confusion. That said, I've been considering adding support for templates that include frontmatter, which would ultimately allow users to specify their own default template for new notes.

**Comment by ajmichels** [comment]
Having more control over the front matter would be much appreciated.

Right now it is either use frontmatter the way the plugin wants you to or when it is disabled the plugin automatically removes it. Neither is ideal. I mean I want to use frontmatter but I want to choose when I use and and what properties are in it.

**Comment by epwalsh** [comment]
> when it is disabled the plugin automatically removes it.

Obsidian.nvim shouldn't be removing existing frontmatter when you have `disable_frontmatter=true`. If this is happening to you please open a bug report and I'll look into it.

**Comment by ajmichels** [comment]
Has happening but then stopped so it must have been something else. Sorry for the false alarm.
